R/summary.R

```diff
@@ -73,8 +73,9 @@ col_paths <- function(x) {
 #'    rrow("r1", 1, 2, 1, 2), rrow("r2", 3, 4, 2,1)
 #' )
 #' col_paths_summary(tbl3)
-row_paths_summary <- function(x) {
+row_paths_summary <- function(x, verbose = TRUE) {
```

Comment:
I'd like to separate this into a separate issue, rather than covering it here because I feel this change invovles a larger discussion about what we want summaries to do and how many of them we have, as I mentioned in our standup

Reply:
Reverted and opened a separated issue #395